gremlin-go/driver/connection_test.go

```diff
@@ -771,10 +752,7 @@ func TestConnection(t *testing.T) {
 		readUsingAnonymousTraversal(t, g)
 
 		// Drop the graph and check that it is empty.
-		dropGraph(t, g)
-		readCount(t, g, "", 0)
-		readCount(t, g, testLabel, 0)
-		readCount(t, g, personLabel, 0)
+		resetGraph(t, g)
```

Comment:
Does resetting the graph defeat the race condition? I thought we'd still have the same issue where it is concurrently writing to the graph. Perhaps I am missing something.

Reply:
I believe Valentyn added a new graph `gtest` specifically for the connection tests, so they'd be accessing separate graphs from the cucumber suite.

Reply:
`resetGraph` does not remove race condition. 
As I understood we have conflict with execution of connection and cucumber test in parallel, so I introduced gtest to not use `g` or `ggrpaph`. But I can be wrong :)

@xiazcy 👍

Reply:
Ack - I thought it was due to concurrent tests here running. Nice fix. Approved.

Reply:
Or actually, I will wait till our conversation tomorrow to approve.